OpenRA.Game/FileFormats/WavLoader.cs

```diff
@@ -35,23 +35,20 @@ public enum WaveType { Pcm = 0x1, ImaAdpcm = 0x11 }
 
 		bool LoadSound(Stream s)
 		{
-			var type = s.ReadASCII(4);
```

Comment:
Why is the auto-detection using the same function as the loading? No wonder this is broken.